test/test_helper.rb

```diff
@@ -25,7 +25,7 @@
 require "webmock/minitest"
 require_relative './matchers'
 
-Minitest::Reporters.use!
+Minitest::Reporters.use! unless ENV['RM_INFO']
```

Comment:
What is this?

Reply:
I followed jet brains article to get there debugger to work

https://blog.jetbrains.com/ruby/2021/04/improved-minitest-support-action-required/